x/stake/types/delegation.go

```diff
@@ -24,6 +24,32 @@ type delegationValue struct {
 	Height int64
 }
 
+// DelegationWithoutDec defines a delegation without the type sdk.Dec for shares
+//
+// TODO: Remove this type (ref: #2096)
+type DelegationWithoutDec struct {
```

Comment:
This TODO comment is incorrect - we cannot change the shares to something other than a decimal - the shares WILL stay as a decimal - it's only Tokens for the Validator object which will change from Decimal Coins to just regular Coins. 

Also I'm assuming you want this type for presentation value? If so this should probably be `DelegationRest` or something along those lines - confusing because it can't be without decimals - it's just that we're using decimals in a string format instead here.